source/api.rst

```diff
@@ -402,14 +403,15 @@ Update PXC cluster image
 
    .. include:: ./assets/code/api-update-cluster-image-response-json.txt
 
-Pass custom my.cnf during the creation of PXC cluster
------------------------------------------------------
+Pass custom my.cnf during the creation of Percona XtraDB Cluster
+----------------------------------------------------------------
 
 **Description:**
 
 .. code-block:: bash
 
-   Create a custom config map containing the contents of the file my.cnf to be passed on to the pxc containers we they are created
+   Create a custom config map containing the contents of the file my.cnf to be
+   passed on to the Percona XtraDB Cluster containers we they are created
```

Comment:
`to the Percona XtraDB Cluster containers we they are created`
probably should be `to the Percona XtraDB Cluster containers we are creating` or something similar.